cmd/maya-apiserver/spc-watcher/handler.go

```diff
@@ -139,3 +147,40 @@ func (c *Controller) getSpcResource(key string) (*apis.StoragePoolClaim, error)
 	}
 	return spcGot, nil
 }
+
+func syncSpc(spcGot *apis.StoragePoolClaim) error {
+	if len(spcGot.Spec.Disks.DiskList) > 0 {
+		// TODO : reconciliation for manual storagepool provisioning
+		return fmt.Errorf("no reconciliation for manual provisioning for storagepoolclaim %s", spcGot.Name)
+	}
+	glog.V(1).Infof("Syncing storagepoolclaim %s", spcGot.Name)
+	// Get kubernetes clientset
+	// namespaces is not required, hence passed empty.
+	newK8sClient, err := k8s.NewK8sClient("")
+	if err != nil {
+		return err
+	}
+	// Get openebs clientset using a getter method (i.e. GetOECS() ) as
+	// the openebs clientset is not exported.
+	newOecsClient := newK8sClient.GetOECS()
+
+	// Get the current count of provisioned pool for the storagepool claim
+	spList, err := newOecsClient.OpenebsV1alpha1().StoragePools().List(metav1.ListOptions{LabelSelector: string(apis.StoragePoolClaimCPK) + "=" + spcGot.Name})
+	if err != nil {
+		return fmt.Errorf("unable to list csp:%v", err)
```

Comment:
Re-frame the error for storagepools and not csp.

Reply:
fixed